src/components/admin.tsx

```diff
@@ -759,21 +762,25 @@ function Admin({ user }: adminProps) {
     hours = DEFAULT_SELF_DESTRUCT_HOURS
   ) => {
     if (navigator.share) {
-      setIsSettingAssignLink(true);
       try {
         await navigator.share({
           title: title,
           text: body,
           url: url
         });
+        setSelectedPostal(postalcode);
```

Comment:
Bugfix here to address issue where every postal code indicator activates on click of the assign/personal button.